gbpservice/neutron/nsf/config_agent/firewall.py

```diff
@@ -0,0 +1,154 @@
+import sys
+import ast
+
+from oslo_config import cfg
+from oslo_messaging import target
+from oslo_log import log as logging
+from neutron import manager
+
+from neutron_fwaas.db.firewall import firewall_db
+
+from gbpservice.neutron.nsf.config_agent import RestClientOverUnix as rc
+from gbpservice.neutron.nsf.config_agent import topics
+from neutron.common import rpc as n_rpc
+
+LOG = logging.getLogger(__name__)
+
+Version = 'v1'  # v1/v2/v3#
+
+
+class Firewall(object):
```

Comment:
All this code can be abstracted into an RpcClient class (similar to RpcAgent in main.py). And can be used in loadbalancer.py, vpn.py.